app/lib/clients/create_employees.rb

```diff
@@ -5,6 +5,26 @@ def initialize(employees, time_off)
     @dates = create_dates
   end
 
+  def add_employee(employee)
+    EmployeeEvents.create!(external_id: employee['id'], display_name: employee['displayName'],
+                           birthday: employee['birthday'], hire_date: employee['hireDate'],
+                           is_photo_uploaded: employee['isPhotoUploaded'], photo_url: employee['photoUrl'],
+                           time_off_employees: find_time_off_employees)
```

Comment:
This method is named `add_employee` but it creates an `EmployeeEvent`. That seems surprising to me.